lib/go-tc/server_capability.go

```diff
@@ -0,0 +1,28 @@
+package tc
```

Comment:
Based on the file naming convention in this PR, should this file be named `server_server_capability.go`?